partcad-cli/src/partcad_cli/click/commands/lint.py

```
@@ -0,0 +1,76 @@
+import partcad as pc
+from partcad.context import Context
+from partcad.user_config import user_config
+from partcad.lint.all import get_linting_checks
+from partcad_cli.click.cli_context import CliContext
+
+
+async def cli_lint_async(ctx: Context, packages: list[str], filter_prefix: str) -> None:
+    tasks = []
+    lint_checks = get_linting_checks(user_config.threads_max)
+    if filter_prefix:
+        lint_checks = list(filter(lambda l: l.name.startswith(filter_prefix), lint_checks))
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)